modules/mobile/app/controllers/mobile/v0/veterens_affairs_eligibility_controller.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+module Mobile
+  module V0
+    class VeterensAffairsEligibilityController < ApplicationController
```

Comment:
I was trying to be consistent with naming since we spelled out Community Cares rather then CC but maybe VA is a well known enough short hand to be used?

Reply:
👍

Reply:
Oops! good catch.